pkg/test/resourcefixture/testdata/basic/sql/v1beta1/sqlinstance/sqlinstance-auditconfig-direct/_http.log

```diff
@@ -555,8 +557,7 @@ User-Agent: kcc/controller-manager
 
 {
   "databaseVersion": "SQLSERVER_2022_EXPRESS",
-  "instanceType": "CLOUD_SQL_INSTANCE",
-  "maintenanceVersion": "SQLSERVER_2022_EXPRESS_CU12_GDR.R20240501.00_05",
```

Comment:
Why is maintenanceVersion no longer being passed?

Reply:
It's not in the user's specified, desired state. This was probably incorrect before.